SimG4CMS/Calo/interface/CaloGVHit.h

```diff
@@ -88,7 +93,8 @@ class CaloGVHitLess {
 class CaloGVHitEqual {
 public:
   bool operator()( const CaloGVHit* a, const CaloGVHit* b) {
-    return (a->getTrackID()     == b->getTrackID() && 
+    return (a->getEventID()     == b->getEventID() && 
+	    a->getTrackID()     == b->getTrackID() && 
 	    a->getUnitID()      == b->getUnitID()  && 
 	    a->getDepth()       == b->getDepth()   &&
 	    a->getTimeSliceID() == b->getTimeSliceID());
```

Comment:
Only `inline` implementations are allowed in a header file: [4.3](https://github.com/cms-sw/cms-sw.github.io/blob/master/cms_coding_rules.md#4----technical-coding-rules-1)

Reply:
@cvuosalo Are you suggesting that there should be an `inline` keyword, or that the function definition should be moved to `.cc`?

Reply:
I am using inline keyword now

Reply:
The keyword is redundant in this context, we have tons of code without the keyword, and 4.3 does not explicitly require the use of the keyword.

Reply:
Yes, any function implemented inside a class definition is automatically inline.

Reply:
I wanted to make sure all of the functions in CaloGVHit.h were intended as inline functions. Some of them are a bit long to be inline, but I guess they are fine like this.